lldb/include/lldb/lldb-types.h

```diff
@@ -42,12 +42,15 @@ typedef void *rwlock_t;
 typedef void *process_t;                          // Process type is HANDLE
 typedef void *thread_t;                           // Host thread type
 typedef void *file_t;                             // Host file type
-typedef unsigned int __w64 socket_t;              // Host socket type
+typedef uintptr_t socket_t;                       // Host socket type
 typedef void *thread_arg_t;                       // Host thread argument type
 typedef unsigned thread_result_t;                 // Host thread result type
 typedef thread_result_t (*thread_func_t)(void *); // Host thread function type
 typedef void *pipe_t;                             // Host pipe type is HANDLE
 
+// printf macro for file_t
+#define PRIuFT PRIuPTR
+
```

Comment:
Could we remove this part. It's technically a part of our public API. I'd recommend using llvm::formatv and LLDB_LOG (not LLDB_LOGF) to format these. It should not have this issue as it doesn't require a type specification.

Reply:
I'm trying to fix Windows warnings at the minute, I'll try this.

Reply:
https://github.com/llvm/llvm-project/pull/146867

Reply:
Thanks.